test/job.template.yaml

```diff
@@ -58,7 +58,7 @@ spec:
         resources:
           limits:
             cpu: 0.5
-            memory: 300Mi
+            memory: 350Mi
```

Comment:
The cause of the OOMKilled issue I noticed when trying to investigate the original issue. I tried using older versions of Julia and still had the same problem so most likely a package dependency increased memory usage. Without access to the original logs it's hard to track down what dependencies had changed.